spec/features/edit_page_spec.rb

```diff
@@ -35,4 +35,26 @@
 
     expect(page).to have_content("true")
   end
+
+  it "displays selectable strings as dropdowns", :js do
+    customer = create(:customer, kind: :standard)
+
+    visit edit_admin_customer_path(customer)
+
+    select "vip", from: "Kind"
+
+    click_on "Update Customer"
+
+    expect(page).to have_content("KIND")
+    expect(page).to have_content("vip")
+
+    click_on "Edit"
```

Comment:
It seems like everything below this point is testing the same functionality as above. Can we limit it to only one form submit?